### PR TITLE
[ADD] website_crm_privacy_policy addon

### PR DESCRIPTION
**Pull request by antespi**
This addon was written to add a checkbox to contact us form for user privacy policy validation

Privacy policy is provided by Website legal page addon (website_legal_page, merge pending at https://github.com/OCA/website/pull/45)

### CODE REVIEW

File: website_crm_privacy_policy/README.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Privacy policy is provided by Website legal page addon (website_legal_page)
+
+Usage
+=====
+
+Website editors can change easily any text of these pages using website builder
+
+
+Credits
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/README.rst:31]: Add bug tracker section

File: website_crm_privacy_policy/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Python source code encoding : https://www.python.org/dev/peps/pep-0263/
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/__init__.py]: Remove header

File: website_crm_privacy_policy/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        'website',
+        'website_crm',
+        'website_legal_page',
+    ],
+    'data': [
+        'views/assets.xml',
+        'views/templates.xml',
+    ],
+    'author': 'Trey, '
+              'Antiun Ingeniería S.L.',
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/__openerp__.py]: Add `Odoo Community Association (OCA)`

File: website_crm_privacy_policy/static/src/js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    $(document).ready(function() {
+        var _t = openerp._t;
+
+        // Validate form
+        $('form[action="/crm/contactus"]').on('submit', function(e) {
+            // Validate privacy_policy is checked
+            if(!$('input[name="privacy_policy"]').is(':checked')) {
+                e.preventDefault();  // Prevent form from submitting
+                alert(_t('You must accept our Privacy Policy.'));
+                // alert(_t('Es obligatorio aceptar la Política de Privacidad.');
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/static/src/js/scripts.js]: Remove this line

File: website_crm_privacy_policy/views/templates.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <!-- Privacy Policy -->
+        <template id="privacy_policy" inherit_id="website_crm.contactus_form">
+            <xpath expr="//form/div[@class='form-group'][last()]" position="before">
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/views/templates.xml]: Why not using `<form position="inside"` or any other simpler marker?

File: website_crm_privacy_policy/views/templates.xml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+                                <input type="checkbox" name="privacy_policy" required="True"/>
+                                I have read and accept the
+                                <a href="/page/privacy" title="Privacy Policy">Privacy Policy</a>.
+                            </label>
+                        </div>
+                    </div>
+                </div>
+            </xpath>
+            <xpath expr="//form/div/div/button" position="attributes">
+                <attribute name="class">btn btn-primary btn-block</attribute>
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/views/templates.xml]: Why do you need this?

File: website_crm_privacy_policy/static/src/js/scripts.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+ * #    GNU Affero General Public License for more details.
+ * #
+ * #    You should have received a copy of the GNU Affero General Public License
+ * #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * #
+ * ##############################################################################
+ */
+
+// Check jQuery available
+if (typeof jQuery === 'undefined') { throw new Error('POS with tax Addon requires jQuery') }
```
Comment (OSguard) [inline website_crm_privacy_policy/static/src/js/scripts.js]: Error message unclear

**Comment by antespi** [comment]
@pedrobaeza, @OSguard any news? we have this addon in production in several customers running OK.
Thanks for reviewing

**Comment by rafaelbn** [comment]
:+1:

**Comment by OSguard** [comment]
Test fail!

**Comment by antespi** [comment]
Hi @OSguard, those errors are unrelated and all PRs in this repo are affected. I've done a rebase and you can check it in runbot. This addon just add a checkbox in contactus form. 
Thanks for your review

**Comment by rafaelbn** [comment]
Checked :+1:

File: website_crm_privacy_policy/README.rst [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+This module was written to add a checkbox to contact us form for user privacy
+policy validation
+
+Privacy policy is provided by Website legal page addon (website_legal_page)
+
+Usage
+=====
+
+Website editors can change easily any text of these pages using website builder
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/README.rst:15]: Add new "Try me on runbot" button

File: website_crm_privacy_policy/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Website CRM privacy policy',
+    'category': 'Website',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/__openerp__.py]: Use new version numbering scheme: 8.0.1.0.0

**Comment by pedrobaeza** [comment]
Don't add original SVG files (you're providing the link to them), but only the composed one.

File: website_crm_privacy_policy/README.rst (hunk 8 above)
Comment (antespi) [inline website_crm_privacy_policy/README.rst:15]: webite repo_id is missing in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt and I can't find it in runbot. Could you update the list?

Comment (pedrobaeza) [inline website_crm_privacy_policy/README.rst:15]: OK, I'll do it. Anyways, the repo ID is 186

Comment (pedrobaeza) [inline website_crm_privacy_policy/README.rst:15]: https://github.com/OCA/maintainer-tools/pull/108 for the inclusion of the repo ID

**Comment by antespi** [comment]
@pedrobaeza comments fixed, thanks for your review.

**Comment by pedrobaeza** [comment]
Unrelated Travis error, so :+1:

File: website_crm_privacy_policy/__openerp__.py [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Website CRM privacy policy',
+    'category': 'Website',
+    'version': '1.0.0',
```
Comment (pedrobaeza) [inline website_crm_privacy_policy/__openerp__.py]: This is still not `8.0.1.0.0`. See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Comment by rafaelbn** [comment]
As test error are unrelated and all PRs in this repo are affected could you @OSguard review again?
I have tested and not find any errors.
Thanks

**Comment by pedrobaeza** [comment]
:+1:

**Comment by JavierIniesta** [comment]
Tested on runbot.

:+1:

**Comment by pedrobaeza** [comment]
Travis error unrelated (fixed in progress in https://github.com/OCA/website/pull/75), so I merge.

**Pull request merged**